### ISSUE TITLE
BBC Radio now missing meta tags, downloads work but show as "failed"

### ISSUE THREAD

**Issue by naq59** [Ascoware/get-iplayer-automator#133]
Thanks first of all for the 2 new versions but I´m now having issues downloading BBC Radio 4 Extra content. The info is missing which then causes problems trying to index  in iTunes and although the download works, a dialog message appears with "Download failed - reason unknown". That file then has "dash" added to the file name. The log says dyld:Symbol not found 
 Am I missing something here or is just a temporary issue with the BBC site?

Safari 11.0.3  OSX 10.11.6

**Comment by skovatch** [comment]
Can you be more specific about the show you are downloading? A dyld message indicates something missing in GiA, or one of the libraries it uses.

Try downloading Just A Minute, which I know I tried myself and could download that.

**Comment by naq59** [comment]
20 minutes ago I tried A Charles Paris Mystery part 2, Men from the Ministry, Paul Scott - The Raj Quartet, Book at Bedtime Ian Sansom episode 2, 15 Minute Drama Based on a true story# 2. All of them the same and Men from the Ministry would´t download at all at first. I downloaded other programmes yesterday, including Just a Minute, which was fine but trying it now again results in the same issues as with the others "Problem unknown", no info but download is complete. Log contains:INFO: Skipping all versions of this programme
dyld: lazy symbol binding failed: Symbol not found: 
I noticed the version of "ffmpeg was built for OSX 10.13" and I´m on 10.11.2 - could this be causing any issues?
